webapp/components/file_attachment.jsx

```diff
@@ -120,7 +114,7 @@ class FileAttachment extends React.Component {
             var re3 = new RegExp('\\)', 'g');
             var url = fileUrl.replace(re1, '%20').replace(re2, '%28').replace(re3, '%29');
```

Comment:
Yep, that's fine. I'll be in here in a few days with the file table stuff anyway so I'll probably get it then if it's still bugging me.